### ISSUE TITLE
series labels are not working for candle stick chart

### ISSUE THREAD

**Issue by RahulVishK** [apache/echarts#18743]
### Version

5.4.2

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor.html?c=candlestick-simple&code=PYBwLglsB2AEC8sDeAoWsAeBBDEDOAXMmurACYCGYFRA2gOQBMADAIwDsAtK854wCz0ANLCZsuPPgFZholh269GANlliFkxu3oBdEgF8hJAJ458RJIZJ4ApgCcINwrFol0qUqTDGQNovQBjCmgyABsnSACAa2E3UkpqOjjPWhYRAGZ-ER4MgA4dI08U_mYMqQzS2ClmAuTSWnTWPIz0kX5-WqL69Nzs8qq2xj0uzqLQigAjG1DYAg8u2DwAC2AAdyIwOwBXG1hChYAzYDsAWyowewIAIiWIO6u6_Tin9D19AG4gA

### Steps to Reproduce

1 create a basic candle stick chart 
2 give "label" option to candle series 

### Current Behavior

series label are not shown in candle stick series 

https://echarts.apache.org/examples/en/editor.html?c=candlestick-simple&code=PYBwLglsB2AEC8sDeAoWsAeBBDEDOAXMmurACYCGYFRA2gOQBMADAIwDsAtK854wCz0ANLCZsuPPgFZholh269GANlliFkxu3oBdEgF8hJAJ458RJIZJ4ApgCcINwrFol0qUqTDGQNovQBjCmgyABsnSACAa2E3UkpqOjjPWhYRAGZ-ER4MgA4dI08U_mYMqQzS2ClmAuTSWnTWPIz0kX5-WqL69Nzs8qq2xj0uzqLQigAjG1DYAg8u2DwAC2AAdyIwOwBXG1hChYAzYDsAWyowewIAIiWIO6u6_Tin9D19AG4gA

### Expected Behavior

Hello!
 It appears that the candlestick chart in the charting library you are using does not have built-in support for series labels. While the documentation may mention series labels for bar series or line series, this functionality is not available specifically for candlestick charts.

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by helgasoft** [comment]
> this functionality is not available specifically for candlestick charts

Correct, no labels for _candlestick_.  But it has _markPoint_ which has a label 🎉
[Demo Code](https://echarts.apache.org/examples/en/editor.html?c=candlestick-simple&code=CYQwLiAEC8kNoChLJcuByATABgIwHYBaXbQzAFnQBpIcaBmcmkhgDivVwGcx0BdKklQoMOAsVKYArNUjlsDKQwWQpCrADtg_QcJFY8REmQBss-rjYN6Nck3T0ATtoFC9owxLL5z7SLiVkQPJMDnIwAAt-N0g-AG4EABsQACMuGEhQCAA6AFsQAAcACgKYAD5IAG8hRwBTMABXRw0qyA0QXNqALgK4KQFIAGMAe2HnLvhe7AGSuFw-AGpezD4ASgB6FcgAXwTt1YRhgrAAS2GW2GrkAA8AQWuTrgnKyDAATwLu9EHwWoBzMZvdDbXSQN73R7PEFCMCjRKnApdF7Q5BZEBcerPSBcYZNQbdTLgKAo7G1RwnWpPeAxK7Cd6fCbfEBaRKU06DADW1BiyFqGhGwAJtL01wmKjeXTmVFCNnIrj0JNQ-UcHIACsMThowM8eSguG9cilhokAMonABeBJUuuQyRStUSSMgBWGXBOp3OjNZADNeDRvWN8mAwGSuuhKiltsDQXo0RM7VwaDbsSdWVqJmBHA1ajGUNsYvnkHwELsgA)
![image](https://github.com/apache/echarts/assets/13038071/5e02ed21-b5cb-413e-87ae-0c28fc0e6dba)
NB: please close issue if problem solved.

**Comment by RahulVishK** [comment]
Hi @helgasoft 
Thank you for this suggestion. As, suggested we have tried you're suggestion it is working for single label case. In our case we need to show multiple labels. We have multiple markpoints?. 
Thank you
